vizro-core/tests/unit/vizro/models/_navigation/test_nav_bar.py

```diff
@@ -73,11 +74,21 @@ def test_items_with_with_pages_icons(self, pages_as_dict):
 class TestNavBarBuildMethod:
     """Tests NavBar model build method."""
 
+    common_args = {"offset": 6, "withArrow": True, "zIndex": 1, "position": "bottom-start"}
```

Comment:
Can we actually move all of these defaults to CSS instead of having it inside the arguments of the dmc component?

Reply:
I don't think these are CSS styling arguments though? They are [component properties](https://www.dash-mantine-components.com/components/tooltip) and not part of `style`.

Reply:
Interesting, I am wondering why these are actually not part of style, because offset/position can be CSS properties: https://developer.mozilla.org/en-US/docs/Web/CSS/offset-position

If this is not possible, I guess users should create custom components of the `NavLink` when they want to style their tooltip differently in terms of offset/position. 

@nadijagraca , if you could just ensure that these are aligned with the designs, that would be great! E.g. we should stick with the design rule that paddings/margins should always be multiples of 4.